ChiTech/ChiPhysics/PhysicsMaterial/transportxsections/prop_transportxs_01b_readfromchi.cc

```diff
@@ -470,26 +482,27 @@ void chi_physics::TransportCrossSections::
       auto& f = file;
       auto& fw = first_word;
 
-      if (fw == "SIGMA_T_BEGIN")    Read1DXS ("SIGMA_T"  , sigma_tg  , f, num_groups, ln, ls);
-      if (fw == "SIGMA_F_BEGIN")    Read1DXS("SIGMA_F"   , sigma_fg  , f, num_groups, ln, ls);
+      if (fw == "SIGMA_T_BEGIN")    Read1DXS ("SIGMA_T"  , sigma_t  , f, num_groups, ln, ls);
+      if (fw == "SIGMA_F_BEGIN")    Read1DXS("SIGMA_F"   , sigma_f  , f, num_groups, ln, ls);
+      if (fw == "SIGMA_A_BEGIN")    Read1DXS("SIGMA_A"   , sigma_a  , f, num_groups, ln, ls);
       if (fw == "NU_BEGIN")         Read1DXS("NU"        , nu        , f, num_groups, ln, ls);
       if (fw == "NU_PROMPT_BEGIN")  Read1DXS("NU_PROMPT" , nu_prompt , f, num_groups, ln, ls);
       if (fw == "NU_DELAYED_BEGIN") Read1DXS("NU_DELAYED", nu_delayed, f, num_groups, ln, ls);
-      if (fw == "CHI_PROMPT_BEGIN") Read1DXS("CHI_PROMPT", chi_g     , f, num_groups, ln, ls);
-      if (fw == "CHI_BEGIN")        Read1DXS("CHI_"      , chi_g     , f, num_groups, ln, ls);
-      if (fw == "DDT_COEFF_BEGIN")  Read1DXS("DDT_COEFF" , ddt_coeff , f, num_groups, ln, ls);
+      if (fw == "CHI_PROMPT_BEGIN") Read1DXS("CHI_PROMPT", chi     , f, num_groups, ln, ls);
+      if (fw == "CHI_BEGIN")        Read1DXS("CHI_"      , chi     , f, num_groups, ln, ls);
```

Comment:
Aside from the duplicate association of ```chi``` to total and prompt, might the trailing underscore in the string ```CHI_``` prevent the call to ```Read1DXS("CHI_"      , chi     , f, num_groups, ln, ls);``` from functioning as expected?

Reply:
Fixed. Great catch.